json-rpc/types/src/views.rs

```diff
@@ -465,6 +465,18 @@ pub struct TransactionView {
     pub vm_status: VMStatusView,
     pub gas_used: u64,
 }
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq)]
+pub struct TransactionsWithProofsView {
+    pub first_transaction_version: Option<u64>,
```

Comment:
I wonder what is the case we get first_transaction_version is None, could you write a test for it?